modules/express/test/unit/lightning/lightningInvoiceRoutes.test.ts

```diff
@@ -68,7 +68,7 @@ describe('Lightning Invoice Routes', () => {
 
       const result = await lightningRoutes.handleCreateLightningInvoice(req);
 
-      should(result).deepEqual(expectedResponse);
+      should(result).deepEqual(Invoice.encode(expectedResponse));
```

Comment:
From the client's perspective, you should test that the response can be decoded.